### DISCUSSION TITLE
About html

### DISCUSSION THREAD

**Discussion by Sia2010** [ossillate-inc/packj#22] [Q&A] [answered]
When I run the example in README, it shows that "HTML summary available at: /tmp/packj_audit_drftpk4_/report_gls1a533.html".
I wonder how to find this content-the html web?

**Comment by ashishbijlani** [comment, marked as answer]
You can view the final report in your browser by specifying the path: file:///tmp/packj_audit_drftpk4_/report_gls1a533.html
